tests/test_video_segmentation_views.py

```
@@ -0,0 +1,535 @@
+import pytest
+from django.test import TestCase, override_settings
+from django.urls import reverse
+from rest_framework.test import APIClient
+from rest_framework import status
+from unittest.mock import Mock, patch
+from pathlib import Path
+import tempfile
+import os
+
+from endoreg_db.models import VideoFile, Label, LabelType, LabelVideoSegment, VideoPredictionMeta
+from endoreg_db.models.administration.person import Gender
```

Pylint (3.3.7)

[error] No name 'Gender' in module 'endoreg_db.models.administration.person'

(E0611)